RobotCode/src/main/java/frc/robot/Commands/ShooterCommand.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Subsystems.ShooterSubsystem;
+
+public class ShooterCommand extends CommandBase
+{
+	// makes the subsystem object
```

Comment:
The declaration does not make the shooter object. The object is passed into the constructor and we simply store the value